user_guide.md

```diff
@@ -9,7 +9,7 @@ This guide will walk you through the basics of deploying and interacting with Ku
 ## Requirements
  * Kubernetes >= 1.8 [see here](https://github.com/kubeflow/tf-operator#requirements)
  * ksonnet version [0.9.2](https://ksonnet.io/#get-started) or later. (See [below](#why-kubeflow-uses-ksonnet) for an explanation of why we use ksonnet)
-
+ * An existing 2-node (at least) kubernate cluster. Nodes need to have storage >= 20 GB due to the ML libraries and third party packages being bundled in Kubeflow Docker images 
```

Comment:
@inc0 after looking at the logs, my wording should be more specific to CPUs instead of nodes. I used a AWS `t2.medium` node for going through the kubeflow user guide.

If the node had had more than 2 CPUs, spawning would have been fine. `t2.medium` was documented to have 2 'vCPU', but it just wasn't enough for spawning another pod.

Please let me know if I should update the wordings or I safely can close the PR, assuming they are common prerequisites in Kubernetes/Jupyter worlds.

```
Name:         jupyter-mimi
Namespace:    kubeflow
Node:         <none>
Labels:       app=jupyterhub
              component=singleuser-server
              heritage=jupyterhub
              hub.jupyter.org/username=mimi
Annotations:  <none>
Status:       Pending
IP:           
Containers:
  notebook:
    Image:  gcr.io/kubeflow-images-staging/tensorflow-1.7.0-notebook-gpu:v20180403-1f854c44
    Port:   8888/TCP
    Args:
      start-singleuser.sh
      --ip="0.0.0.0"
      --port=8888
      --allow-root
    Requests:
      cpu:     1
      memory:  1G
    Environment:
      JUPYTERHUB_API_TOKEN:           X
      JPY_API_TOKEN:                  X
      JUPYTERHUB_CLIENT_ID:           user-mimi
      JUPYTERHUB_HOST:                
      JUPYTERHUB_OAUTH_CALLBACK_URL:  /user/mimi/oauth_callback
      JUPYTERHUB_USER:                mimi
      JUPYTERHUB_API_URL:             http://tf-hub-0:8081/hub/api
      JUPYTERHUB_BASE_URL:            /
      JUPYTERHUB_SERVICE_PREFIX:      /user/mimi/
      MEM_GUARANTEE:                  1G
      CPU_GUARANTEE:                  1.0
    Mounts:
      /var/run/secrets/kubernetes.io/serviceaccount from no-api-access-please (ro)
Conditions:
  Type           Status
  PodScheduled   False 
Volumes:
  no-api-access-please:
    Type:        EmptyDir (a temporary directory that shares a pod's lifetime)
    Medium:      
QoS Class:       Burstable
Node-Selectors:  <none>
Tolerations:     node.alpha.kubernetes.io/notReady:NoExecute for 300s
                 node.alpha.kubernetes.io/unreachable:NoExecute for 300s
Events:
  Type     Reason            Age                From               Message
  ----     ------            ----               ----               -------
  Warning  FailedScheduling  1m (x37 over 11m)  default-scheduler  No nodes are available that match all of the predicates: Insufficient cpu (2), PodToleratesNodeTaints (1).
```

Reply:
It would be good to specify minimum requirements about number of CPUs.